x-pack/plugins/reporting/public/lib/reporting_api_client/reporting_api_client.test.ts

```diff
@@ -52,10 +52,7 @@ describe('ReportingAPIClient', () => {
     it('should send a delete request', async () => {
       await apiClient.deleteReport('123');
 
-      expect(httpClient.delete).toHaveBeenCalledWith(
-        expect.stringContaining('/delete/123'),
-        expect.any(Object)
-      );
+      expect(httpClient.delete).toHaveBeenCalledWith(expect.stringContaining('/delete/123'));
```

Comment:
the delete endpoint is no longer called with the option object that has `asSystemRequest`